modules/UpgradeWizard/uw_utils.php

```diff
@@ -759,6 +759,28 @@ function upgradeUWFiles($file) {
         $allFiles[] = "$from_dir/include/utils/autoloader.php";
     }
 
+	if(file_exists("$from_dir/include/UploadFile.php")) {
+		$allFiles[] = "$from_dir/include/UploadFile.php";
+	}
+	if(file_exists("$from_dir/include/SugarTheme/SugarTheme.php")) {
+		$allFiles[] = "$from_dir/include/SugarTheme/SugarTheme.php";
+	}
+
+	// add extra files to post upgrade process
+	if(file_exists(realpath("$from_dir/../scripts/files_to_add_post.php"))) {
+		include(realpath("$from_dir/../scripts/files_to_add_post.php"));
+		if(isset($filesToAddPost) && is_array($filesToAddPost) && $filesToAddPost) {
+			foreach($filesToAddPost as $file) {
+				if(file_exists("$from_dir/$file")) {
+					$allFiles[] = "$from_dir/$file";
+					$GLOBALS['log']->info("File added to post upgrade: $from_dir/$file");
```

Comment:
OK `/` works on Windows and Linux today, but I prefer to keep it 100% unbreakable on every OS, and future proof. This function does the job:
```
function file_build_path() {
    return join(DIRECTORY_SEPARATOR, func_get_args($segments));
}
```
Usage:   `$pathname = file_build_path($from_dir, $file);`